solidity/test/bridge/VendingMachine.test.ts

```diff
@@ -1,25 +1,33 @@
-const { expect } = require("chai")
-const {
-  increaseTime,
+import { ethers, waffle, helpers } from "hardhat"
+import { expect } from "chai"
+import type { Signer } from "ethers"
+import { vendingMachineDeployment, constants } from "../fixtures"
+// eslint-disable-next-line import/extensions
+import type { TestERC20, TBTC, VendingMachine } from "../../typechain"
+
+import {
   to1e18,
   to1ePrecision,
   ZERO_ADDRESS,
```

Comment:
We can now use `const ZERO_ADDRESS = ethers.constants.AddressZero`, see [here](https://github.com/keep-network/keep-core/blob/874e247e8f21cec976f93b38604c37a1f77b8872/solidity/random-beacon/test/RandomBeacon.Relay.test.ts#L24)

Reply:
Done.